metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -938,9 +938,12 @@ ControllerResult<Map<Uuid, ApiError>> deleteTopics(ControllerRequestContext cont
                 BoundedList.newArrayBacked(MAX_RECORDS_PER_USER_OP, ids.size());
         for (Uuid id : ids) {
             try {
+                log.info("Starting deletion of topic with ID {}.", id);
                 deleteTopic(context, id, records);
                 results.put(id, ApiError.NONE);
+                log.info("Successfully deleted topic with ID {}.", id);
             } catch (ApiException e) {
+                log.error("Failed to delete topic with ID {} due to API exception.", id, e);
```

Comment:
This isn't an error. It should be at INFO level. But more importantly, shouldn't all the topics deleted in this RPC share a single log message?